apps/src/assetManagement/IconList.jsx

```diff
@@ -0,0 +1,84 @@
+var IconListEntry = require('./IconListEntry.jsx');
+var icons = require('./icons').aliases;
+var msg = require('../locale');
+
+/**
+ * A component for managing icons.
+ */
+module.exports = React.createClass({
+  propTypes: {
+    assetChosen: React.PropTypes.func.isRequired,
+    search: React.PropTypes.string.isRequired
+  },
+
+  getMatches: function (query) {
+    var letters = query.split(''), results = {};
```

Comment:
I don't necessarily mind the pattern of declaring multiple variables with a single `var`, but it's not really one we use for the most part in our code base. For that reason, I'd prefer that we split this out, i.e.
```
var letters = query.split('');
var results = {};
```
Same with line 48